source/templates/writing-helpers.md

```diff
@@ -59,11 +59,11 @@ You may also find yourself rendering your view classes in multiple
 places using the `{{view}}` helper. In this case, you can save yourself
 some typing by registering a custom view helper.
 
-For example, let’s say you have a view called `App.CalendarView`.
+For example, let’s say you have a view called `CalendarView`.
```

Comment:
Technically default exports don't have a name, so you have a `views/calendar-view.js` or more generically a `calendar view`. L66/68 are ok, but I'd say `Using the calendar view …`.
I'll probably start a style guide for documentation contributors and then normalize all these references accordingly.